packaging/addons/openproject/bin/postinstall

```diff
@@ -63,16 +63,19 @@ rake_commands="db:migrate db:seed"
 # set rails_cache_store
 memcached_servers="$(${CLI} config:get MEMCACHED_SERVERS || echo "")"
 if [ -z "$memcached_servers" ]; then
-  ${CLI} config:set RAILS_CACHE_STORE=file_store
+  ${CLI} config:set OPENPROJECT_RAILS__CACHE__STORE=file_store
 else
-  ${CLI} config:set RAILS_CACHE_STORE=memcache
+  ${CLI} config:set OPENPROJECT_RAILS__CACHE__STORE=memcache
 fi
 
+${CLI} config:unset RAILS_CACHE_STORE
+
 # create attachments folder
-attachments_path=$(${CLI} config:get ATTACHMENTS_STORAGE_PATH || echo "/var/db/${APP_NAME}/files")
+attachments_path=$(${CLI} config:get OPENPROJECT_ATTACHMENTS__STORAGE__PATH || ${CLI} config:get ATTACHMENTS_STORAGE_PATH || echo "/var/db/${APP_NAME}/files")
```

Comment:
Documentation needs to be updated too: `docs/installation-and-operations/configuration/README.md:286` has a paragraph about running `openproject config:get ATTACHMENTS_STORAGE_PATH` to customize the path.

Also, I do not know if `OPENPROJECT_ATTACHMENTS_STORAGE_PATH` should be supported too.

Reply:
There are other references to ATTACHMENTS_STORAGE_PATH:
* `docs/installation-and-operations/misc/migration/README.md`
* `packaging/scripts/backup`
* and 2 others involving docker in `docker/prod/Dockerfile` and `docker/prod/entrypoint.sh`

Reply:
Fixed in b91ec14.